jupyterlab/labbox_ephys_widgets_jp/src/pluginWidgets.tsx

```diff
@@ -193,7 +163,7 @@ const PluginComponentWrapper: FunctionComponent<PluginComponentWrapperProps> = (
     model.on('change:curation', () => {
       if (!curationSubfeed) {
         curationDispatch({
-          type: 'SetCuration',
+          type: 'SET_CURATION',
```

Comment:
Use all caps to be consistent with ADD_RECORDING, DELETE_RECORDINGS, etc. These are the actions stored in the subfeed associated with the workspace.